types/src/parse.d.ts

```diff
@@ -1,3 +1,15 @@
-import { ColorConstructor } from "./color";
+import { PlainColorObject } from "./color";
```

Comment:
I think that the `ColorConstructor` type is right here. Here are the type definitions for both types:

<https://github.com/LeaVerou/color.js/blob/64430d9e56a025cd6dea750d7e0073caf4c37eec/types/src/color.d.ts#L31-L41>

Note that `PlainColorObject` has the key `space`, while `ColorConstructor` has the key `spaceId`.

Reply:
I found it a bit annoying that the `parse` function doesn't return a `PlainColorObject`. I currently use `getColor` for parsing because it does the `spaceId` to `space` fixup and is typed to return a `PlainColorObject`.

Reply:
Not sure what distinction this is making, since the actual `Color` constructor accepts both `spaceId` and `space`. 

But `parse()` can definitely be made to return an object with both `space` and `spaceId`, I suppose I just wanted to avoid the duplication. This is orthogonal to this PR though

Reply:
I will revert to using ColorConstructor for now though, since that's more accurate, and we can discuss how to fix the broader issue after this PR is merged